src/v/cluster/types.h

```diff
@@ -574,6 +574,7 @@ struct incremental_topic_updates
     static constexpr int8_t version_with_initial_retention = -7;
     static constexpr int8_t version_with_write_caching = -8;
     static constexpr int8_t version_with_iceberg_property = -9;
+    static constexpr int8_t version_with_remote_read_and_write = -10;
```

Comment:
i think we can stop updating this version and adl serialisation code, all updates with the property will already be serde serialised. Can we add a comment the updating the ADL serialization code is not longer needed ?

Reply:
Removed ADL updates, added comment about deprecation.